app/src/main/java/org/bspb/smartbirds/pro/ui/utils/FormsConfig.java

```diff
@@ -0,0 +1,34 @@
+package org.bspb.smartbirds.pro.ui.utils;
+
+import org.bspb.smartbirds.pro.R;
+
+public enum FormsConfig {
+    species_class(
+            new String[]{"birds", "herptiles", "mammals", "invertebrates", "plants"},
+            new int[]{R.string.class_birds, R.string.class_herptiles, R.string.class_mammals, R.string.class_invertebrates, R.string.class_plants}
+    ),
+    threats_primary_types(
+            new String[]{"threat", "poison"},
+            new int[]{R.string.primary_type_threat, R.string.primary_type_poison}
+    ),
+    threats_poisoned_types(
+            new String[]{"dead", "alive", "bait"},
+            new int[]{R.string.poisoned_type_dead, R.string.poisoned_type_alive, R.string.poisoned_type_bait}
+    );
```

Comment:
Seems there are enums that contain the directly used string - can extract the strings as constants or have separate enums for each config that can provide the value and string.id for each item